mixer/adapter/stackdriver/contextgraph/workload.go

```diff
@@ -151,24 +158,60 @@ func (wi workloadInstance) Reify(logger adapter.Logger) ([]entity, []edge) {
 	return []entity{wiEntity, owner, workload}, edges
 }
 
+func (s service) Reify() entity {
+	return entity{
+		containerFullName: fmt.Sprintf("//cloudresourcemanager.googleapis.com/projects/%s",
+			s.istioProject),
+		typeName: "io.istio.Service",
+		fullName: fmt.Sprintf("//istio.io/projects/%s/meshes/%s/services/%s/%s",
+			s.istioProject,
+			url.QueryEscape(s.meshUID),
+			url.QueryEscape(s.namespace),
+			url.QueryEscape(s.name)),
+		location: "global",
+		shortNames: [4]string{
+			url.QueryEscape(s.meshUID),
+			url.QueryEscape(s.namespace),
+			url.QueryEscape(s.name),
+			"",
```

Comment:
Why this extra "" in the short names?
There are expected to be 3 components in the short names.

Reply:
It's a static 4 element array, not a slice. It should not be added to the entity present assertion due to this line of code:

https://github.com/istio/istio/blob/a4065c5bde3ea50e584be4fced7ef1a7e31f036b/mixer/adapter/stackdriver/contextgraph/send.go#L77

Are you seeing it come through?

Reply:
The error I see in the API is that the number of short names in the istio service is != 3.

Reply:
what's the resolution here?  @jeffmendoza should we just make it:

```golang
shortNames: [3]string{
  url.QueryEscape(s.meshUID),
  url.QueryEscape(s.namespace),
  url.QueryEscape(s.name)
},
```

?

Reply:
Looks like a config issue. We should merge this PR now.